kitsune/sumo/static/sumo/js/tests/form-wizard-tests.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from "chai";
+import { FormWizard } from "sumo/js/form-wizard";
+
+describe("form-wizard custom element", () => {
+  let wizard;
+  let slot;
+
+  beforeEach(() => {
+    $("body").empty().html(`
+            <form-wizard>
+                <div name="first">
+                    <p>This is the first step.</p>
+                </div>
+                <div name="second">
+                    <p>This is the second step.</p>
+                </div>
+            </form-wizard>
+        `);
+    wizard = document.querySelector("form-wizard");
+    slot = wizard.shadowRoot.querySelector('slot[name="active"]');
+  });
+
+  it("should render a form-wizard custom element", () => {
+    expect(wizard).to.exist;
+    expect(wizard).to.be.an.instanceof(FormWizard);
+  });
+
+  it("should show the first step by default", () => {
+    let assignedElements = slot.assignedElements();
+    let activeStep = assignedElements[0];
+    expect(assignedElements.length).to.equal(1);
+    expect(activeStep.getAttribute("name")).to.equal("first");
+    expect(activeStep.textContent.trim()).to.equal("This is the first step.");
+  });
+
+  it("should show a different step when the active step changes", () => {
+    let nextSteps = [
+      {
+        name: "first",
+        status: "done",
+      },
+      {
+        name: "second",
+        status: "active",
```

Comment:
@mikeconley curious what you think about possible statuses. It seems like we might actually have five possible states/statuses for form steps (though still tbd if we'll actually allow clicking back and forth between steps in the switching devices wizard):

- active
- completed, can be navigated back to
- completed, can't be navigated back to
- not completed, can be navigated to
- not completed, can't be navigated to

Reply:
Per our last meeting with UX, it sounds like we're not going to support going back through the wizard for now. So I think the states are:

- active
- completed
- not yet unavailable

Reply:
Actually disregard this - probably moot since we're not going to be navigating back and forth between steps. I updated the `<form-wizard>` to only rely on `activeStep` for now in order to start with the simplest possible implementation.